backend/flask_api.py

```diff
@@ -232,14 +232,9 @@ def update_project(project_id):
                 print("Successfully replaced content: ", file=sys.stderr)
                 print(manifest, file=sys.stderr)
                 return make_response('Success')
-            else:
-                v = validator.iter_errors(manifest)
-                if v is not None:
-                    validation_error = []
-                    for error in sorted(validator.iter_errors(manifest), key=str):
-                        print(error.message, file=sys.stderr)
-                        validation_error.append(error.message)
-                raise ApiException("Validation Error: \n" + str(is_valid), 400, validation_error)
+            elif validator.iter_errors(manifest) is not None:
+                validation_errors = [error for error in sorted(validator.iter_errors(manifest))]
+                raise ApiException("Validation Error: \n" + str(is_valid), 400, validation_errors)
```

Comment:
this still calls validator.iter_errors(manifest) twice, what I meant is:

```
validation_errors = [error for error in sorted(validator.iter_errors(manifest))]
if validation_errors:
    raise ApiException("Validation Error: \n" + str(is_valid), 400, validation_errors)
```
but I don't know how costly the call is, so I don't mind it this way